Fmaraev/Hometask_3/task_2.py

```diff
@@ -0,0 +1,7 @@
+st = input('Enter a sentence')
+nst = ''
+for i in st:
+    if nst.find(i) == -1:
+        nst += i
+nst_2 = nst.replace(' ', '')
```

Comment:
why can't we do it before creating nst string? in your solution we should go through every space in given string in for cycle